pkg/datapath/linux/node.go

```diff
@@ -1191,6 +1275,26 @@ func (n *linuxNodeHandler) NodeValidateImplementation(nodeToValidate nodeTypes.N
 	return n.nodeUpdate(nil, &nodeToValidate, false)
 }
 
+// NodeNeighDiscoveryEnabled returns whether node neighbor discovery is enabled
+func (n *linuxNodeHandler) NodeNeighDiscoveryEnabled() bool {
+	return n.enableNeighDiscovery
+}
+
+// NodeNeighborRefresh is called to refresh node neighbor table
+func (n *linuxNodeHandler) NodeNeighborRefresh(nodeToRefresh nodeTypes.Node) error {
```

Comment:
This doesn't need to return an error, because it currently will only return nil.